tests/specs/mcd/verification.k

```diff
@@ -116,12 +116,6 @@ module LEMMAS-MCD [symbolic, kore]
 
     rule { BA1:Bytes +Bytes (BA2:Bytes +Bytes BA3:Bytes) #Equals BA4:Bytes +Bytes BA3 } => { (BA1 +Bytes BA2) #Equals BA4 } [simplification]
 
-    rule { BA1:Bytes +Bytes BA2:Bytes #Equals BA3:Bytes +Bytes BA4:Bytes } =>
```

Comment:
This one and the other removed lemmas for this specialised file were causing a slowdown when interacting with the ones added by this PR.